integration-tests/simnode/src/chain/picasso.rs

```diff
@@ -81,15 +64,15 @@ async fn _parachain_info_storage_override_test(
 	node: &Node<ChainInfo>,
 ) -> Result<(), Box<dyn Error>> {
 	// sudo account on-chain
-	let sudo = node.with_state(None, sudo::Pallet::<picasso_runtime::Runtime>::key).unwrap();
+	let _sudo = node.with_state(None, sudo::Pallet::<picasso_runtime::Runtime>::key).unwrap();
 
 	// gotten from
 	// hex::encode(&parachain_info::ParachainId::<Runtime>::storage_value_final_key().to_vec());
 	let key = hex::decode("0d715f2646c8f85767b5d2764bb2782604a74d81251e398fd8a0a4d55023bb3f")?;
 
 	let raw_key_value: Option<u32> = node.with_state(None, || storage::unhashed::get(&key[..]));
 
-	assert_eq!(raw_key_value, Some(2087));
+	assert_eq!(raw_key_value, Some(2104));
```

Comment:
Is this suposed to be picasso parachain id?

Reply:
Let me revert that, used that while testing